.changelog/35123.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Fixed an issue where **pre-commit** and **validate** were executed on irrelevant untracked files during contribution use case.
```

Comment:
The suggestion is not following one of the demiso-sdk changlog [templates](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs), What about this?
```diff
- - description: Updated **pre-commit** and **validate** commands to use a specific file's list during the contribution flow instead of relying on Git diff.
+ - description: Modified **pre-commit** and **validate** commands to use a specific file's list during the contribution flow instead of relying on Git diff.
```